src/content/blog/2023-07-24-odh-operator-v2.0-blog.md

```diff
@@ -0,0 +1,97 @@
+---
+type: blog
+author: VaishnaviHire
+title: Exploring the DataScienceCluster: A New Custom Resource in OpenDataHub(incubation)
```

Comment:
This colon is causing issues. You will have to escape it. I think you can just wrap the whole thing in quotes or use a slash. Not sure exactly what works but one of them will.